### DISCUSSION TITLE
Rangliste

### DISCUSSION THREAD

**Discussion by henrybrink** [SE-TINF22B2/G5-DuoGradus#10] [Anforderungen]
Damit die Motivation der Nutzer hoch bleibt soll für den Wettbewerb unter den Nutzern eine Rangliste bereitgestellt werden.
Dabei soll es eine globale Rangliste und eine Freundes-Rangliste geben, wobei letztere wichtiger für den Nutzer ist.

**Comment by henrybrink** [comment]
Dafür benötigen wir zudem ein System bei dem ein Nutzer eine Freundschaftsanfrage an einen anderen Nutzer senden kann, diese akzeptiert werden können und Benutzer ihre Freunde auch bearbeiten können.

**Reply by benedictweis** [reply]
Hauptsache kein Live Chat 🙃
